swag-chart/templates/accounts_deploy.yml

```diff
@@ -0,0 +1,56 @@
+{{- with .Values.accounts }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .deploymentName }}
+spec:
+  replicas: {{ .replicas }}
+  selector:
+    matchLabels:
+      app: {{ .app }}
+  template:
+    metadata:
+      labels:
+        app: {{ .app }}
+    spec:
+      containers:
+      - name: {{ .deploymentName }}
+        image: {{ .image }}:{{ .tag }}
+        env:
+          - name: spring.config.import
+            valueFrom:
+              configMapKeyRef:
+                name: env-vars
+                key: spring.config.import
+          - name: eureka.client.service-url.defaultZone
+            valueFrom:
+              configMapKeyRef:
+                name: env-vars
+                key: eureka.client.service-url.defaultZone
+          - name: email.host
+            valueFrom:
+              configMapKeyRef:
+                name: env-vars
+                key: email.host
+          - name: email.port
+            valueFrom:
+              configMapKeyRef:
+                name: env-vars
+                key: email.port
+          - name: email.username
+            valueFrom:
+              configMapKeyRef:
+                name: env-vars
+                key: email.username
+          - name: email.Password
+            valueFrom:
+              configMapKeyRef:
+                name: env-vars
+                key: email.Password
+        resources:
+          limits:
+            memory: {{ quote .memoryLimit }}
+            cpu: {{ quote .cpuLimit }}

```

Comment:
we need requests on everything, not just limits

Reply:
done.